docs/admin/authentication.md

```diff
@@ -33,7 +33,9 @@ plane, must authenticate when making requests to the the API server.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
+authenticate API requests through authentication plugins. A bearer token
+must be something that can be included in an HTTP header value without any
```

Comment:
the sentence "A bearer token must be something that can be included in an HTTP header value..." just seems out of place here, and still sort of vague. I'd rather tweak this text in the bearer token auth method sections to make it clearer the literal token is passed in the header:

> When using token authentication from an http client the API server expects an `Authorization`
header with a value of `Bearer SOMETOKEN`.